### PR TITLE
Libmesh submodule update

### PR DESCRIPTION
**Pull request by roystgnr**
- Bounding box testing for .msh $Entities in GmshIO::read(), to detect
  file inconsistencies
- Added Netgen as an optional submodule.  With Netgen enabled, libMesh
  now includes a `NetGenMeshInterface` class which can be used to
  generate tetrahedral meshes within triangulated boundaries or within
  volume mesh domains.
- Added `libmesh/fuzzy_equals.h` methods: `absolute_fuzzy_equals()`
  and `relative_fuzzy_equals()`, for checking equality to within a
  specified absolute or relative tolerance.  Added `l1_norm()` and
  `l1_norm_diff()` methods as necessary to classes (`TypeVector`,
  `NumericVector`, `SparseMatrix`) and subclasses which might be
  compared.
- Fixes for `make distcheck` testing of release candidate tarballs
- Updates to EIM error indicator normalization in Reduced Basis code

### CODE REVIEW

File: modules/doc/content/newsletter/2024/2024_07.md [hunk 1]
```diff
@@ -58,6 +58,24 @@ for a complete description of all MOOSE changes.
 - `System::is_initialized()` is now `const`
 - Added `LibmeshPetscCall` macro that can wrap any PETSc function call without need for `LIBMESH_CHKERR(ierr)`
 
+### `2024.07.27` Update
+
+- Bounding box testing for .msh $Entities in GmshIO::read(), to detect
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2024/2024_07.md:63]: If for some reason you have to push again, I might suggest moving the `LibmeshPetscCall` item down to this new update

**Review by lindsayad — COMMENTED** [review]
👍

File: modules/doc/content/newsletter/2024/2024_07.md (hunk 1 above)
Comment (roystgnr) [reply modules/doc/content/newsletter/2024/2024_07.md:63]: Yeah; I'm still not sure how I got the news item in the previous update but not the actual commit.  I usually just `git log --merges hash..` and expand+reorder what I read there.  I wonder if I somehow managed to `git checkout origin/master` right before the `LibmeshPetscCall` merge and `git log --merges hash..origin/master` right after.

Comment (lindsayad) [reply modules/doc/content/newsletter/2024/2024_07.md:63]: It's because I suggested that you add it to the last newsletter entry. So it was my flub

**Review by lindsayad — APPROVED** [review]

**Review by milljm — APPROVED** [review]
for my part, Conda looks good to me.

**Pull request merged**